pkg/vere/mdns.c

```diff
@@ -0,0 +1,220 @@
+/// @file
+
+#include "vere.h"
+
+#include "dns_sd.h"
+
+typedef struct _mdns_payload {
+  mdns_cb*      cb;
+  DNSServiceRef sref;
+  char*         who;
+  char*         our;
+  uint16_t      port;
+  uv_poll_t*    poll;
```

Comment:
It looks like `poll` always has the same lifetime as `payload`, so it can just be `uv_poll_t` (no `*`).

Reply:
Done